Server/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1617,3 +1620,11 @@ int CLuaPedDefs::TakeAllWeapons(lua_State* luaVM)
     lua_pushboolean(luaVM, false);
     return 1;
 }
+
+bool CLuaPedDefs::IsPedReloadingWeapon(CPed* const ped)
+{
+    if (ped)
+        return ped->IsReloadingWeapon();
+
+    return false;
```

Comment:
what about `return ped ? ped->IsReloadingWeapon() : false;`?

Reply:
`ped && ped->IsReloadingWeapon()`
But how could `ped` ever be null?
It can't be, unless it's called from somewhere else but the parser

Reply:
You're right, I'll use a direct check of isReloadingWeapon.